src/main/java/org/smartregister/pathevaluator/action/ActionHelper.java

```diff
@@ -91,6 +102,29 @@ public static ResourceType getResourceType(Action action) {
 			case FAMILY:
 			case PERSON:
 				return clientDao.findClientById(entity);
+			case TASK:
+				List<QuestionnaireResponse.Item> items = questionnaireResponse.getItem();
```

Comment:
check if questionnaireResponse is null

Reply:
QuestionnaireResponse cannot be null here. There's a check before this call and an operation on L96